gridcapa-task-manager-app/src/main/java/com/farao_community/farao/gridcapa/task_manager/app/TaskManagerController.java

```diff
@@ -23,6 +20,7 @@
  */
 @Controller
 @RequestMapping
+@CrossOrigin(origins = "*")
```

Comment:
Is that necessary ? I think it is better to avoid it if we can

Reply:
no it's my bad . it's a forgotten test hack